terraform/modernisation-platform-account/iam.tf

```diff
@@ -219,3 +219,9 @@ resource "aws_iam_access_key" "member-ci" {
     create_before_destroy = true
   }
 }
+
+# temporary users for healthcheck
```

Comment:
If this block is commented out the whole module is disabled, right?
Might be worth putting a comment here for that. Rather than someone butchering the code to disable the temp access.